web/src/scripts/containers/TabbedEditor.jsx

```diff
@@ -302,6 +306,7 @@ const mapStateToProps = (state, ownProps) => {
       showInvisibles: state.preferences[CATEGORIES.EDITOR][PREFERENCES.EDITOR.SHOW_INVISIBLES],
       styleActiveLine: state.preferences[CATEGORIES.EDITOR][PREFERENCES.EDITOR.HIGHLIGHT_ACTIVE_LINE],
       showIndentGuides: state.preferences[CATEGORIES.EDITOR][PREFERENCES.EDITOR.SHOW_INDENT_GUIDES],
+      npmRegistry: state.preferences[CATEGORIES.EDITOR][PREFERENCES.EDITOR.NPM_REGISTRY],
```

Comment:
This `options` object is specifically for CodeMirror text editor options. Can you move `npmRegistry` a few lines up into the parent object `return {...}`?